.github/ISSUE_TEMPLATE/bug_feature.md

```diff
@@ -0,0 +1,16 @@
+---
+name: Bug Report or Feature Request
+about: Report a bug or suggest a new feature!
+title: ""
+labels: ""
+assignees: ""
+---
+
+### Problem
+
+<!--
+For a bug report, please copy and paste any error messages from the application or command-line here.
+For a feature request, Please state how the new functionality could potentially benefit the community.
+-->
+
```

Comment:
Inspired by the way conda-forge instructs users. `<!--, -->` remains invisible so the user does not have to delete this.